detect_interval: 3

# detect model param
detect:
  conf_thres: 0.5
  model_half: True
  model_path: None
  device: 'gpu' # 'gpu' or 'cpu'

# camera param
camera:
  image_shape: [1280,720]
  camera_fps: 30
  # camera_dev  and camera_param_root
  far_camera:
    dev: 0
    path: 'camera1'
  near_camera:
    dev: 4
    path: 'camera2'